### PR TITLE
Mesh pts fixes for updated wids

### PR DESCRIPTION
**Pull request by m-alperen-sener**
Updating wid 500 and 7 according to latest PTS updates.
tester PR: https://github.com/zephyrproject-rtos/zephyr/pull/63872

### CODE REVIEW

File: autopts/pybtp/btp/mesh.py [hunk 1]
```diff
@@ -1008,11 +1008,18 @@ def mesh_composition_data_get(net_idx, addr, page):
...
-    data = bytearray(struct.pack("<HHB", net_idx, addr, page))
-
-    (rsp,) = iutctl.btp_socket.send_wait_rsp(*MESH['composition_data_get'], data)
-
-    (status,) = struct.unpack_from('<B', rsp)
+    req_page = page
+
+    while req_page >= 0:
+        data = bytearray(struct.pack("<HHB", net_idx, addr, req_page))
+        (rsp,) = iutctl.btp_socket.send_wait_rsp(*MESH['composition_data_get'], data)
```
Comment (PavelVPV) [inline autopts/pybtp/btp/mesh.py:1015]: Add `Response parameters:` to `btp_mesh_node.txt`?

Comment (m-alperen-sener) [reply autopts/pybtp/btp/mesh.py:1015]: added (Y)

File: doc/btp_mesh_node.txt [hunk 2]
```diff
@@ -226,6 +226,8 @@ Commands and responses:
 		Command parameters:     Network index (2 octets)
 		                        Node address (2 octets)
 		                        Page (1 octet)
+		Response parameters:	Received Page (1 octet)
+								Composition Data (variable length)
```
Comment (PavelVPV) [inline doc/btp_mesh_node.txt:230]: Extra tabs?

Comment (m-alperen-sener) [reply doc/btp_mesh_node.txt:230]: Same number of tabs; I am not sure why it looks weird here. 
image

Comment (m-alperen-sener) [reply doc/btp_mesh_node.txt]: changed the tabs with space as the rest of the file has

**Review by PavelVPV — APPROVED** [review]

**Comment by m-alperen-sener** [comment]
@sjanc can you have a look at that, it is needed for latest PTS release.

**Review by sjanc — APPROVED** [review]

**Pull request merged**